Makefile.am

```diff
@@ -24,11 +24,13 @@ SUBDIRS = \
   mc \
   $(NEUTRINORDPDIR) \
   libxrdp \
+  librfxcodec \
```

Comment:
Users who checkout xrdp, don't get librfxcodec by default. They would get a broken experience. That's really too big of a topic to be part of "miscellaneous fixes"